README.md

```diff
@@ -5,9 +5,27 @@
 
 Search Insights lets you report click, conversion and view metrics using the [Algolia Insights API](https://www.algolia.com/doc/rest-api/insights/#overview).
 
+## Table of Contents
+
+<!-- toc -->
+
+- [Getting started](#getting-started)
+  - [On Browser](#on-browser)
```

Comment:
The TOC needs to be updated and we're good!

Reply:
Good point!